test/src/sdk-harness/Forc.lock

```diff
@@ -4,9 +4,9 @@ dependencies = []
 
 [[package]]
 name = 'sdk-harness'
-dependencies = ['std git+https://github.com/fuellabs/sway?tag=v0.10.3#e7676db6f4ebc7a3885f87514d16a703a99410d7']
+dependencies = ['std git+https://github.com/fuellabs/sway?tag=v0.11.0#e7676db6f4ebc7a3885f87514d16a703a99410d7']
```

Comment:
In a separate PR, can you fix this to use a relative-path `std`, then once it's merged into `master` pick up the changes here?